src/redux/actions/actions.js

```diff
@@ -1,26 +1,47 @@
 import * as actionTypes from './actionTypes';
 
-export function GetPosts() {
+export function getUsersAndPosts() {
   return function(dispatch) {
-    fetch('https://jsonplaceholder.typicode.com/posts')
-      .then(response => response.json())
-      .then(json => {
-        dispatch({
-          type: actionTypes.SAVE_POSTS,
-          postsData: json.slice(-10).sort((a, b) => a.id < b.id),
-        });
-      });
+    const usersData = getUsers();
+    const postsData = getPosts();
+    // eslint-disable-next-line no-undef
+    Promise.all([usersData, postsData]).then(data =>
+      linkPostsAndUsers(data, dispatch),
+    );
   };
 }
 
-export function GetUsers() {
-  return function(dispatch) {
-    fetch('https://jsonplaceholder.typicode.com/users')
-      .then(response => response.json())
-      .then(usersData => {
-        dispatch({ type: actionTypes.SAVE_USERS, usersData });
-      });
-  };
+export function linkPostsAndUsers(data, dispatch) {

```

Comment:
I don't think linking is a good idea here.
At first it might look strait-forward and easy to use but as long as your app gets bigger it becomes disaster.
Let's imagine situation when user data gets update.
You need to update it in `userData` and `postsData`
Store should be a singe source of truth and it you should avoid data duplication as much as possible.

Please take a look at this article. I hope it will help to understand what I mean
https://hackernoon.com/shape-your-redux-store-like-your-database-98faa4754fd5